subgraph/schema.graphql

```diff
@@ -0,0 +1,41 @@
+type Asset @entity(immutable: true) {
+  id: Bytes!
+  address: Bytes!
+  decimals: Int!
+  label: String!
+}
+
+type Offer @entity {
+  id: Bytes!
+  domainName: String!
+  sourceAsset: Asset!
+  destAsset: Asset!
+  offerAddress: Bytes!
+  domainOwner: Bytes!
+  depositHistory: [BigInt!]!
```

Comment:
Maybe deposit history should be a separate entity - so Offer itself can be immutable. Total sum of deposits can be dynamically computed. Offer can just store initial deposit.

This can be modified later

Reply:
Sounds good